cloud.conf.json.template

```diff
@@ -0,0 +1,12 @@
+{
+    "default": {
```

Comment:
Please, lets use some reference to cloud 6, 7 and cloud 7 crowbar init.
Also, we need 3 different ports for the ssh connection on each machine